### PR TITLE
[devtools] add wndt71

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?
- add "What's New In DevTools (Chrome 71)"

**Fixes:** N/A

**Target Live Date:** 2018-10-10

- [ ] This has been reviewed and approved by @jbingham
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by kaycebasques** [comment]
localhost_8080_web_updates_2018_10_devtools

File: src/content/en/updates/2018/10/devtools.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+<figure>
+  <img src="/web/updates/images/2018/10/focus4.png"
+       alt="The blue border remains even though the page doesn't really have focus."/>
+  <figcaption>
+    <b>Figure 7</b>. The blue border remains even though the page doesn't really have focus
+  </figcaption>
+</figure>
+
+## Initiator and priority information now in HAR imports and exports {: #HAR }
+
```
Comment (jbingham) [inline src/content/en/updates/2018/10/devtools.md]: Can we mention "HTTP Archive (HAR)" once in the text for people who don't know the acronym?

File: src/content/en/updates/2018/10/devtools.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    <b>Figure 12</b>. How to show the <b>Priority</b> column
+  </figcaption>
+</figure>
+
+Note: The `_initiator` and `_priority` fields begin with underscores because the [HAR spec][spec]{: .external }
+states that custom fields must begin with underscores.
+
+[spec]: https://dvcs.w3.org/hg/webperf/raw-file/tip/specs/HAR/Overview.html
+
+## Access the Command Menu from the Main Menu {: #command-menu }
```
Comment (jbingham) [inline src/content/en/updates/2018/10/devtools.md]: We should also let people know about the keyboard shortcut that already existed. Eg, something like "The Command Menu provides a list of all available Devtools panels, tabs, and features. It used to only be accessible by typing  ..., and not everyone knew the keyboard shortcut. Now you can access it directly from the Main Menu." Or more conversationally, we could say, "Did you know about the [...] keyboard shortcut? It shows the Command Menu, a list of... Now you don't need to know the shortcut. The Command Menu appears in the Main Menu."

File: src/content/en/updates/2018/10/devtools.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+<figure>
+  <img src="/web/updates/images/2018/10/store2.png"
+       alt="Store as global variable in the DOM Tree."/>
+  <figcaption>
+    <b>Figure 3</b>. Store as global variable in the DOM Tree
+  </figcaption>
+</figure>
+
+## Emulate page focus {: #focus }
```
Comment (jbingham) [inline src/content/en/updates/2018/10/devtools.md]: "Ever struggle to debug styles that are different when the page has focus? Suppose you've got a collection of buttons..."

**Pull request merged**